tests/framework/installer/install_helper.go

```diff
@@ -308,10 +303,6 @@ func (h *InstallHelper) CleanupCluster(clusterName string) {
 	if err != nil {
 		logger.Errorf("namespace  %s cannot be deleted,err -> %v", clusterName, err)
 	}
-
-	h.k8shelper.WaitUntilPodInNamespaceIsDeleted("rook-ceph-mon", clusterName)
-	h.k8shelper.WaitUntilNameSpaceIsDeleted(clusterName)
-
```

Comment:
Should we add on test where we check to make sure all rook cluster data is deleted? Could be done in a saperate ticket.

Reply:
i was thinking it would be a good thing, but I'm not sure it would be worth it. We're doing the operations to make sure the pods and resources are deleted, we just aren't waiting for kubernetes to kill the pods. And we are stopping and unmounting block devices. That seems like the more interesting test.